lib/gh/merge_commit.rb

```diff
@@ -58,9 +58,9 @@ def has_merge_commit?(hash)
 
     def github_done_checking?(hash)
       case hash['mergeable_state']
-      when 'checking'                               then false
-      when 'unknown'                                then hash['merged']
-      when 'clean', 'dirty', 'unstable', 'stable'   then true
+      when 'checking'                                                  then false
+      when 'unknown'                                                   then hash['merged']
+      when 'clean', 'dirty', 'unstable', 'stable', 'blocked', 'behind' then true
       else fail "unknown mergeable_state #{hash['mergeable_state'].inspect} for #{url(hash)}"
```

Comment:
I'm guessing that the intention was for the `fail` to be handled by the calling code and then surfaced however makes sense at that level, which wasn't happening as expected in our case.